### ISSUE TITLE
Starter set of profiles in Sinopia

### ISSUE THREAD

**Issue by jgreben** [LD4P/sinopia_editor#851]
Work with @kirkhess, et. al. to 
- [ ] get the latest LC profiles to be Sinopia compliant and loaded into Sinopia as starter set.
- [ ] be able to transform newly created LC profiles to work in Sinopia

**Comment by jgreben** [comment]
@kirkhess You mentioned that you might already have a script that could help with this. If so, can you point me to it?

**Comment by ndushay** [comment]
Is this meant to address this problem?

![image](https://user-images.githubusercontent.com/96775/60555671-cbfec180-9cf2-11e9-80d8-d0362ec19967.png)

**Comment by jgreben** [comment]
> Is this meant to address this problem?
> 
> ![image](https://user-images.githubusercontent.com/96775/60555671-cbfec180-9cf2-11e9-80d8-d0362ec19967.png)

Not necessarily. It may as long as the set of resource templates that are loaded are complete with all of their references, and the chosen template is one of those.

**Comment by michelleif** [comment]
awaiting latest LC profiles from kirk

**Comment by jermnelson** [comment]
Testing loading the Profiles from the [verso profile branch](https://github.com/lcnetdev/verso/tree/profile_update) . Corresponding with @kirkhess and will resume testing after he does follow-up commits.

**Comment by michelleif** [comment]
progress is here: https://docs.google.com/spreadsheets/d/1j2OUoXBmmvfC-vCnG02ynoQGFB5Q-pFsnRSszleZ78w/

**Comment by michelleif** [comment]
starter set is now here: https://github.com/LD4P/ld4ptemplates/
and those templates are in DEV for review
keeping ticket open until the profiles are reviewed and loaded to PRD
